frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/LanguageSelection/index.js

```
@@ -1,60 +1,392 @@
+          <img
+            src="https://tfstate8auyj.blob.core.windows.net/egov-dev-assets/currency-coin-rupee.svg"
+            style={{ margin: "auto" }} />
+          Quick Payments</div>
+        <div style={styles.verticalDivider}></div>
+        <div style={styles.card}>  <img
+          src="https://tfstate8auyj.blob.core.windows.net/egov-dev-assets/Vector%20(1).svg"
+        /> Property Tax</div>
+        <div style={styles.verticalDivider}></div>
+        <div style={styles.card}> <img
+          src="https://tfstate8auyj.blob.core.windows.net/egov-dev-assets/water_voc.svg"
+        /> Water Tax</div>
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)